src/python/pants/backend/python/target_types.py

```diff
@@ -86,6 +87,13 @@ class PythonSourceField(SingleSourceField):
 
 class PythonGeneratingSourcesBase(MultipleSourcesField):
     expected_file_extensions: ClassVar[tuple[str, ...]] = ("", ".py", ".pyi")
+    help = generate_multiple_sources_field_help_message(
```

Comment:
Ditto to apply this specifically to `python_test_utils`, `python_sources`, and `python_tests`, rather than once for them all

Reply:
Done!